x/ratelimit/keeper/quota.go

```diff
@@ -0,0 +1,65 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/Stride-Labs/stride/v4/x/ratelimit/types"
+)
+
+// Add new quota to the store
+func (k Keeper) AddQuota(ctx sdk.Context, quota types.Quota) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.QuotaKeyPrefix))
+
+	b := k.cdc.MustMarshal(&quota)
+	store.Set([]byte(quota.Name), b)
+}
+
+// Remove quota from the store
+func (k Keeper) RemoveQuota(ctx sdk.Context, name string) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.QuotaKeyPrefix))
+	store.Delete([]byte(name))
+}
+
+// Get quota from the store
+func (k Keeper) GetQuota(ctx sdk.Context, name string) (quota types.Quota) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.QuotaKeyPrefix))
+
+	b := store.Get([]byte(name))
+	if b == nil {
+		return quota
+	}
+
+	k.cdc.MustUnmarshal(b, &quota)
+	return quota
+}
+
+// Get all quotas from the store
+func (k Keeper) GetAllQuotas(ctx sdk.Context) (list []types.Quota) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.QuotaKeyPrefix))
+	iterator := sdk.KVStorePrefixIterator(store, []byte{})
```

Comment:
just curious - is this the same as doing `store.Iterator(nil, nil)`? (that's what I've seen before)

Reply:
yes, its just same thing.